server-side/package.json

```diff
@@ -7,14 +7,19 @@
   "scripts": {
     "test": "jest",
     "start": "node app.js",
-    "dev": "nodemen app.js"
+    "dev": "nodemon app.js"
+  },
+  "jest": {
```

Comment:
please check if npm test is still running in main after you added this jest block